README.md

```diff
@@ -62,6 +62,21 @@ $ station metrics --follow
 ...
 ```
 
+### `$ station activity`
+
+Get Station activity logs.
+
+```bash
+$ station activity
+[3/14/2023, 10:23:14 AM] Saturn Node is online and connected to 1 peers
+[3/14/2023, 10:23:18 AM] Saturn Node is online and connected to 9 peers
+
+$ station activity --follow
+[3/14/2023, 10:23:14 AM] Saturn Node is online and connected to 1 peers
+[3/14/2023, 10:23:18 AM] Saturn Node is online and connected to 9 peers
+...
+```
```

Comment:
Let's split this into two examples, similar to how you documented `station logs`.